sdk/storage/azblob/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/Azure/azure-sdk-for-go/sdk/storage/azblob
+
+go 1.17
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.16.2
```

Comment:
@jhendrixMSFT @JeffreyRichter do we want to have azblob reference the most recent azcore or are we ok with the older version?

Reply:
Yes, it should reference the latest

Reply:
You will need to regenerate with the `preview.27` version of the code generator.

Reply:
The autorest version we were using earlier was actually penultimate to latest version i.e. `preview.26` which had dependency on `azcore:0.16.2` so we couldn't update it. Now that `preview.27` is available, I've updated the generated code of SDK to the latest.